packages/core/package.json

```diff
@@ -64,11 +66,18 @@
     "react-window": "^1.8.8"
   },
   "devDependencies": {
+    "@playwright/test": "^1.34.3",
+    "@storybook/react": "^6.5.16",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
The tests are failing - `HvCarousel`'s images are role `presentation` instead of `img`.
Might be caused by this bump 🤔